rmgpy/tools/canteramodel.py

```diff
@@ -363,7 +368,12 @@ def plot(self, data, top_species=10, top_sensitive_reactions=10):
                 ReactionSensitivityPlot(x_var=time,
                                         y_var=reaction_sensitivity_data[j * num_ct_reactions:(j + 1) * num_ct_reactions],
                                         num_reactions=top_sensitive_reactions).barplot(
-                    os.path.join(self.output_directory, '{0}_{1}_sensitivity.png'.format(i + 1, species.to_chemkin())))
+                    os.path.join(self.output_directory, '{0}_{1}_reaction_sensitivity.png'.format(i + 1, species.to_chemkin())))
+                if self.thermo_SA:
+                    ThermoSensitivityPlot(x_var=time,
+                                          y_var=thermodynamic_sensitivity_data[j * num_ct_species:(j + 1) * num_ct_species],
+                                          num_species=top_sensitive_species).barplot(
+                        os.path.join(self.output_directory, '{0}_{1}_thermo_sensitivity.png'.format(i + 1, species.to_chemkin())))
```

Comment:
Is Cantera calculate sensitivity wrt enthalpy? Is that is the case, then should we assign the argument `xlabel` to `'dln(c)/d(H_i) [(kcal/mol)^-1]'` (maybe unit based on mass fraction instead?)?

Reply:
So @mjohnson541 and I talked about this once, and I think we decided that thermo sensitivity with respect to Gibbs free energy is exactly the same as with respect to enthalpy. It is completely fair to change the label, but I don't think we need to worry if not.

Reply:
Also, I don't think it is possible to convert this based on mass fraction, as H_i does not depend on the mass fraction. You could use mass based units like `kcal/g` but I don't think this is as useful. We tend to think about the thermochemistry of the species involved in terms of molar units. Correct me if I am wrong though, I don't know if perhaps there is a better way that RMS or someone else uses.